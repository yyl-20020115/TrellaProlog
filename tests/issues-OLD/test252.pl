:-initialization(main).

main :-
	writeq(Ö-Œ = s-t), nl,
	writeq(Ö-Œ = s-t), nl,
	writeq(-'Ö'-'Œ'+(.)+'A'), nl.
