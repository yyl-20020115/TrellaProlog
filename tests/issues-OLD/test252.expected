_3-_4=s-t
_3-_4=s-t
-'Ö'-'Œ'+'.'+'A'
